pkgs/development/web/nodejs/corepack.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, nodejs }:
+
+let
+  inherit (nodejs) version;
+in
+stdenv.mkDerivation {
+  name = "corepack-nodejs-${version}";
+
+  nativeBuildInputs = [ nodejs ];
+
+  unpackPhase = "true";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    corepack enable --install-directory $out/bin
+    # Also wrap npm
+    corepack enable --install-directory $out/bin npm
+  '';
+
+  meta = {
+    description = "Wrappers for npm, pnpm and yarn via nodejs's corepack";
```

nit

```suggestion
    description = "Wrappers for npm, pnpm and Yarn via Node.js Corepack";
```

refs: https://github.com/nodejs/node/blob/cc80513551503397e07c46f84e57ea48c342561e/doc/README.md#L87-L89